src/main/resources/proton.accessWidener

```diff
@@ -0,0 +1,2 @@
+accessWidener   v1  named
+accessible  method net/minecraft/block/Blocks   never   (Lnet/minecraft/block/BlockState;Lnet/minecraft/world/BlockView;Lnet/minecraft/util/math/BlockPos;)Z
```

Comment:
I could've just created a lambda but why not

Reply:
no

Reply:
accesswiderner good.